test/views/declaration/summary/SummaryPageViewSpec.scala

```diff
@@ -186,12 +175,38 @@ class SummaryPageViewSpec extends WordSpec with MustMatchers with ExportsDeclara
       summaryPage must include("supplementary.summary.locations.officeOfExit")
     }
 
-    "has correct items" in {
+    "has correct items" when {
+      "on Standard journey" in {
+
+        summaryPage must include("declaration.itemType.title")
```

Comment:
Are we asserting on the full HTML here?

Reply:
Good point. Absolutely nothing has ids on this page and we would have to use CSS regex to get the items.
We have a story this sprint to address the "Check your answers" page, probably we should address this there.